crates/egui/src/context.rs

```diff
@@ -1050,7 +1050,7 @@ impl Context {
     ///
     /// If the widget already exists, its state (sense, Rect, etc) will be updated.
     #[allow(clippy::too_many_arguments)]
-    pub(crate) fn create_widget(&self, w: WidgetRect) -> Response {
+    pub(crate) fn create_widget(&self, w: WidgetRect, ignore_focus: bool) -> Response {
```

Comment:
Negations in booleans is a source of confusion. Better to invert it and call it `allow_focus`